search/src/Components/upload/uploader_category_mapper.js

```diff
@@ -81,7 +81,7 @@ class CategoryMapper extends React.Component {
                             return (
                                 <li className={category.role && category.name ? classes.row : `${classes.row} ${classes.incomplete}`}>
                                     {/^[0-9A-Fa-f]{6}$/i.test(category.name) ? <div className={classes.color} style={{backgroundColor: '#' + category.name}}></div> : <p>{category.name}</p>}
-                                    <p className={classes.text_field}> is a</p>
+                                    <p className={classes.text_field}>&nbsp;&nbsp;is a</p>
```

Comment:
@ElevnLi we really shouldn't be using `<p>` for things that are only _parts_ of a sentence. We also have style for `classes.text_field` to apply a right margin. Is the right solution to apply a left margin as well?